paasta_tools/cli/cmds/local_run.py

```diff
@@ -712,9 +735,30 @@ def paasta_local_run(args):
         sys.stderr.write("If you meant to pull the docker image from the registry, explicitly pass --pull\n")
         return 1
 
+    try:
+        system_paasta_config = load_system_paasta_config()
+    except PaastaNotConfiguredError:
+        sys.stdout.write(PaastaColors.yellow(
+            "Warning: Couldn't load config files from '/etc/paasta'. This indicates\n"
+            "PaaSTA is not configured locally on this host, and local-run may not behave\n"
+            "the same way it would behave on a server configured for PaaSTA.\n"
+        ))
+        system_paasta_config = SystemPaastaConfig({"volumes": []}, '/etc/paasta')
+
+    local_run_config = system_paasta_config.get_local_run_config()
+
     service = figure_out_service_name(args, soa_dir=args.yelpsoa_config_root)
-    cluster = guess_cluster(service=service, args=args)
-    instance = guess_instance(service=service, cluster=cluster, args=args)
+    if args.cluster:
+        cluster = args.cluster
+    else:
+        try:
+            cluster = local_run_config['default_cluster']
+        except KeyError:
+            sys.stderr.write(PaastaColors.yellow(
+                "PaaSTA on this machine has not been configured with a default cluster.\n"
+                "For now the default cluster is being set to 'testopia'.\n"))
+            cluster = 'testopia'
```

Comment:
This was just here in case the puppet configs didn't ship before the paasta changes. I'll remove it.